snap/snapcraft.yaml

```diff
@@ -1,16 +1,19 @@
 name: butterfly
 grade: stable
 adopt-info: butterfly
-summary: Powerful, minimalistic, cross-platform, opensource note-taking app.
-description: |
```

Comment:
Why do you delete the summary and description?

Reply:
I have explained everything in my commit messages, Kindly read them.
>Added the metafile properly to adopt snap info from there and added various metadata for other store apps(May seem redundant though).